SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -173,16 +173,29 @@ void USpatialStatics::PrintTextSpatial(UObject* WorldContextObject, const FText
 	PrintStringSpatial(WorldContextObject, InText.ToString(), bPrintToScreen, TextColor, Duration);
 }
 
-FString USpatialStatics::GetActorEntityIDString(const AActor* Actor)
+int64 USpatialStatics::GetActorEntityId(const AActor* Actor)
 {
 	if (Actor != nullptr)
```

Comment:
Should this be a check? Do we have valid reasons for wanting to support nullptr?

Reply:
A check seems reasonable